cmd/reva/main.go

```diff
@@ -114,10 +114,8 @@ func main() {
 	}
 
 	client = rhttp.GetHTTPClient(
-		// TODO make insecure configurable
-		rhttp.Insecure(true),
-		// TODO make timeout configurable
-		rhttp.Timeout(time.Duration(24*int64(time.Hour))),
+		rhttp.Insecure(insecure),
```

Comment:
This `insecure` flag denotes if the GRPC connection to the gateway should work without TLS or not. I'm not sure if we should use the same one for the data gateway service.